packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -6,77 +6,104 @@
     <template #description>
       {{ t('settingsPage.notifications.description') }}
     </template>
-    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+    <div class="divide-y border rounded divide-gray-50 border-gray-100">
       <div
-        v-for="({id, title}) in switches"
-        :key="id"
-        class="py-[16px]"
+        v-if="!props.gql.localSettings.preferences.desktopNotificationsEnabled"
+        class="bg-indigo-100 p-[16px] flex justify-between items-center"
+        data-cy="enable-notifications"
       >
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          <label :for="id">{{ title }}</label>
-          <Switch
-            class="mx-[8px]"
-            :value="props.gql.localSettings?.preferences[id] ?? false"
-            :name="id"
-            @update="(value) => updatePref(value, id)"
-          />
-          {{ gql.localSettings }}
-        </h4>
-      </div>
-      <div class="py-[16px]">
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          {{ t('settingsPage.notifications.notifyMeWhenRunCompletes') }}
-        </h4>
-        <div class="pt-[5px]">
-          <Checkbox
-            v-for="({id, label}) in statuses"
-            :id="id"
-            :key="id"
-            v-model="listRef"
-          >
-            <template #label>
-              <label
-                :for="id"
-                class="text-sm text-gray-900"
-              >
-                {{ label }}
-              </label>
-            </template>
-          </Checkbox>
+        <div class="text-indigo-700 font-medium">
+          {{ t('settingsPage.notifications.enableNotificationsLabel') }}
         </div>
-      </div>
-      <div class="py-[16px]">
-        <Button
-          variant="white"
-          size="32"
-          @click="showTestNotification"
+
+        <ButtonDS
```

Comment:
This button should trigger a notification when clicked, same behavior as the enable notifications banner. You can see that button's click handler here

https://github.com/cypress-io/cypress/blob/bdfa347017330ecf06c4f004b30356bba098468a/packages/app/src/specs/banners/EnableNotificationsBanner.vue#L69-L73

Reply:
Added in https://github.com/cypress-io/cypress/pull/26965/commits/36f826b1e3439d2662fe3631dcf52d08b055f824